cloudflare.go

```diff
@@ -40,7 +40,7 @@ func (api *API) makeRequest(method, uri string, params interface{}) ([]byte, err
 	// Replace nil with a JSON object if needed
 	var reqBody io.Reader
 	switch method {
-	case "GET", "DELETE":
+	case "GET":
```

Comment:
I have removed the `switch` method now and replaced it with a simple `if`. I did (silently) break `DeleteDNSRecord` :/